mobile/src/board/components/BoardView/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Board, Player, GameState } from "../../entity";
+
+import PlayerPawnView from "../PlayerPawnView";
+import TileView from "../TileView";
+
+import "./index.css";
+
+interface BoardProps {
+  board: Board;
+}
+
+/**
+ *
+ */
+const BoardView = ({
+  board: {
+    state: { tiles, remainingTile, gameState, players },
+  },
+}: BoardProps) => {
```

Comment:
Prefer

`() => (<div>)`

to 

`() => {
  return (<div>)
}`

As it removes one level of indentation and therefore makes the code a bit easier to read.